CHANGELOG.md

```diff
@@ -1,26 +1,268 @@
+| Clarified entry about optional sample data                       | **Update**        | [#2474][] |
+| Implemented minor doc site updates                               | **Update**        | [#2388][] |
+| Updated search index for the Magento User Guide                  | **Update**        | [#2418][] |
+| Implemented Spectrum redesign for docs                           | **Update**        | [#2386][] |
+| Added extra information in the FAQ for using the image component | **Update**        | [#2383][] |
+| Added FAQ section on how to query different storeviews           | **Update**        | [#2381][] |
+| Updated the Cloud Deploy tutorial                                | **Update**        | [#2319][] |
+| Updated styleguide colors & typography                           | **Update**        | [#2236][] |
+| Removed an unwanted link                                         | **Update**        | [#2202][] |
+| Corrected mispelling in 'Introduction to React'                  | **Bugfix**        | [#2487][] |
+| Fixed Modal/Portal reference doc generation                      | **Bugfix**        | [#2450][] |
+| Fixed content in Cloud Deploy topic                              | **Bugfix**        | [#2398][] |
+
+### Misc
+
+| Description                                                               | Change type | PR                  |
+| ------------------------------------------------------------------------- | ----------- | ------------------- |
+| Bumped `workbox` dependency to 5.1.2                                      | **Update**  | [#2480][]           |
+| Bumped `websocket-extensions` dependency from 0.1.3 to 0.1.4              | **Update**  | [#2465][]           |
+| Bumped `apollo-server` dependency from 2.6.9 to 2.14.2                    | **Update**  | [#2464][]           |
+| Added Lars Roettig to the list of Community Maintainers                   | **Update**  | [#2439][] [#2293][] |
+| Updated pull request template to use valid markdown syntax for checkboxes | **Update**  | [#2434][]           |
+| Updated docker image                                                      | **Update**  | [#2406][]           |
+| Updated pull request template                                             | **Update**  | [#2389][]           |
+| Renamed docker file to get syntax highlighting                            | **Update**  | [#2374][]           |
+| Bumped jquery from 3.4.1 to 3.5.0 in pwa-devdocs project directory        | **Update**  | [#2370][]           |
+| Updated the node version in venia-concept to allow Node >=10.x            | **Update**  | [#2315][]           |
+| Bumped acorn from 5.7.3 to 5.7.4                                          | **Update**  | [#2312][]           |
+| Updated `minimist` dependency to >= 1.2.2                                 | **Update**  | [#2267][]           |
+| Fixed `https-proxy-agent` dependency issue                                | **Bugfix**  | [#2356][]           |
+
+## Known issues
```

Comment:
Let's get with the team to ensure that there are no additional known issues that we should call out.